pkg/genlib/config/config.go

```diff
@@ -1,19 +1,25 @@
 package config
 
 import (
-	"github.com/elastic/go-ucfg/yaml"
+	"errors"
+
 	"io/ioutil"
+	"math"
 	"os"
+
+	"github.com/elastic/go-ucfg/yaml"
 )
 
+var rangeBoundNotSet = errors.New("range bound not set")
+
 type Ratio struct {
 	Numerator   int `config:"numerator"`
 	Denominator int `config:"denominator"`
 }
 
 type Range struct {
-	Min interface{} `config:"min"`
-	Max interface{} `config:"max"`
+	Min *float64 `config:"min"`
```

Comment:
let's parse as `float64` and convert to `int64` with getters (`MinAsInt64`/`MaxAsInt64`) instead of doing [magic](https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/80/files#diff-eeffd89b6f2d2b3db8542f09093ea87c32181bfa90f4dfba8141118acfa0320dL226-L233) when reading the value to guess the original type.